manage/sawtooth_manage/docker.py

```diff
@@ -154,20 +157,23 @@ def start(self, node_config):
         yaml.dump(compose_dict,
                   open(os.path.join(compose_dir, 'docker-compose.yaml'),
                        mode='w'))
+
+        print("Starting: {}".format(node_name))
```

Comment:
Should we want to take it out completely? We could leave it in cluster, but then it wouldn't be accurate timewise (though that isn't as much of a problem if things are running in parallel).